JASP-Engine/JASP/R/commonAnovaBayesian.R

```diff
@@ -1159,7 +1159,10 @@
   if (is.null(descriptivesContainer)) {
     descriptivesContainer <- createJaspContainer(title = gettext("Descriptives"))
     descriptivesContainer$dependOn(c("dependent", "repeatedMeasuresCells"))
-    descriptivesContainer$position <- 9001 # always last
+    
+    if (is.null(options$confidenceIntervalInterval)) # TRUE implies Bayesian
```

Comment:
So this works fine but I'd prefer something like:

```r
.BANOVAdescriptives <- function(jaspResults, dataset, options, errors, analysisType, ready = TRUE, position = 9001) {

  ...

  descriptivesContainer$position <- position
}
```
That way, other/ future analyses could also reuse this function and still easily change the position.

Reply:
OK, made it like that - also dont need to specify it anymore for the freq anova, because it didn't seem to fix the notes issue anyway